File: tools/dbench/answers/python_haiku/s2/m1_oldest.py
lines = open("users.txt").readlines()
best_name = None
best_age = -1
for line in lines:
    parts = line.strip().split()
    if len(parts) == 3:
        try:
            age = int(parts[2])
            if age > best_age:
                best_age = age
                best_name = parts[1]
        except ValueError:
            pass
if best_name:
    print(f"{best_name} {best_age}")

File: tools/dbench/answers/rust_sonnet/s1/m1_oldest.rs
use std::fs;

fn main() {
    let contents = fs::read_to_string("users.txt").unwrap_or_default();
    let mut oldest_name = String::new();
    let mut oldest_age: i64 = i64::MIN;
    let mut found = false;
    for line in contents.lines() {
        let parts: Vec<&str> = line.split_whitespace().collect();
        if parts.len() == 3 {
            if let Ok(age) = parts[2].parse::<i64>() {
                if age > oldest_age {
                    oldest_age = age;
                    oldest_name = parts[1].to_string();
                    found = true;
                }
            }
        }
    }
    if found {
        println!("{} {}", oldest_name, oldest_age);
    }
}
